package.xml

```diff
@@ -8,5 +8,9 @@
   <url type="repository">https://github.com/ros/ros_environment</url>
   <url type="bugtracker">https://github.com/ros/ros_environment/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_core</buildtool_depend>
+
+  <export>
+    <build_type>ament_cmake</build_type>
```

Comment:
Is use of the ament_cmake build type possible when depending only on ament_cmake_core?

Reply:
Yes, the build type is only relevant to the build tool - it is not related to the build system / CMake part.